fedimint-cli/src/lib.rs

```diff
@@ -242,6 +242,11 @@ struct Opts {
     #[arg(long, env = "FM_PASSWORD")]
     password: Option<String>,
 
+    /// Activate more verbose logging, for full control use the RUST_LOG env
+    /// variable
+    #[arg(short = 'v', long)]
```

Comment:
**note:** It's possible to make clap count number of times this was passed and go even more verbose. Not sure if we need it yet.

Reply:
But then we'd need to think about activating the always-muted modules again at some point. It seems much easier to just refer to `RUST_LOG`.

Which logs to put into which verbosity level is a discussion I didn't want to get into.